.config-schema.json

```diff
@@ -38,6 +38,18 @@
             "default": 1000,
             "x-env-variable": "OPENFGA_LIST_OBJECTS_MAX_RESULTS"
         },
+        "allowWriting1_0Models": {
+            "description": "Allow calling WriteAuthorizationModel with models that have 1.0 schema version",
+            "type": "boolean",
+            "default": false,
+            "x-env-variable": "OPENFGA_ALLOW_WRITING_1_0_MODELS"
+        },
+        "allowEvaluating1_0Models": {
+            "description": "Allow calling Check, Expand, (......) with models that have 1.0 schema version",
```

Comment:
> Allow calling APIs with models that have 1.0 schema version.

Reply:
Updated to list all APIs that are affected